src/style.css

```diff
@@ -1,3 +1,75 @@
+html {
+  --link-default: black;
+  --link-active: #00acdc;
```

Comment:
good practice to  declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector
```css
:root {
  --link-default: black;
  --link-active: #00acdc;
}
```